src/NativeBridge/dllmain.cpp

```diff
@@ -114,11 +114,11 @@ bp::dict pxCall(bp::dict& params)
         else
             retCode = exec(&env, s_graph.c_str(), 0, NULL);
 
+        if (retCode == -1)
+            throw std::runtime_error(env.GetErrorMessage());
```

Comment:
When this is -1 then the `GetData()` call will be skipped. Is this okay since it looks like the `PythonObject` instances which are owned by NativeBridge get deleted only if `GetData()` is called?

Reply:
Great point! I ll put it back

---
In reply to: [326818751](https://github.com/microsoft/NimbusML/pull/273#discussion_r326818751) [](ancestors = 326818751)